src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java

```diff
@@ -33,4 +36,78 @@ public static LocalDate transformFormDateIntoLocalDate(String formFieldName, Str
             throw new FormFieldValidationException(String.format("%s must be a valid date", formFieldName));
         }
     }
+
+    public static List<String> getCommaSeparatedValueFromOcrDataField(String commaSeparatedString) {
+        //TODO clarify what the delimiter is - comma followed by space?
+        //TODO modify tests if delimiter changes
+        //TODO trim empty spaces or count as warning - perhaps incorrectly scanned so needs flagged?
+        if (commaSeparatedString.isEmpty()) {
+            return Collections.emptyList();
+        }
+        return Splitter.on(", ").splitToList(commaSeparatedString);
```

Comment:
I'd suggest to split it by "," (without space) and then:
1. trim all elements from the list
2. remove empty elements

but maybe Jeremy as a BS has better idea.

Reply:
I had a brief chat with Jeremy just before the holidays, and it seems like we want to be very strict - that is also reflected in the ticket: If a checklist contains _any other values_ (even if some of them are valid) then we should throw a warning/error.
The reasoning behind this is apparently if the wrong page or even form gets scanned, or one is missing, or perhaps the image is distorted and OCR couldn't pick the values up, we want to flag that up and not accept potentially invalid values.

Pinged Jeremy for his input as well.